### DISCUSSION TITLE
Any news for the Tailwind 3 UI ?

### DISCUSSION THREAD

**Discussion by RSickenberg** [Okipa/laravel-table#102] [Ideas]
Hello, 

Thanks again for the V5 it's a ground-breaking change.

Do you have any news about the `tailwind` portage?

Regards

**Comment by Okipa** [comment]
Hi @RSickenberg, I'm currently stabilising the v5 and I still don't work on projects based on Tailwind.
As so, it will be difficult for me to setup a Tailwind UI version right now.

Adding this when mastering Tailwind should be quite simple (There is a good example to follow with the Bootstrap 4 implementation) and it may be a good opportunity for Hacktoberfest candidates to submit a full tested pull request about this.

**Comment by flemzord** [comment]
Hey @Okipa 
Any news for support Tailwindcss ?

**Reply by Okipa** [reply]
Hi @flemzord,

Yes I have planned to implement this as Jetstream does use Tailwind by default.
I was on vacation and I now have some time to work on this package.
There are some small bug fixes/features to implement before but I think I'll have something in april.

**Reply by flemzord** [reply]
Awesome!
